### PR TITLE
chore: add new auto lock period values and migrate users if needed

### PR DESCRIPTION
**Pull request by fedeerbes**
# 🔘 PR Type

- Enhancement

# 📜 Background
This PR adds new auto lock period options and migrates users from older values (1' or 10') to new low value 15' if needed. Default value now is 30'

Issue Link: #[ENG-3527](https://linear.app/xverseapp/issue/ENG-3527/change-auto-lock-timer-choices-%5Bext-%F0%9F%8C%8D%5D)

Impact:
- Auto lock flow, setting and lock/unlock logic

# 🖼 Screenshot / 📹 Video
| NORMAL FLOW | SIMULATED UPDATE |
| :---: | :---: |
| <video src='https://github.com/secretkeylabs/xverse-web-extension/assets/18606335/faa89abc-64c1-420f-a7e5-685215ac3cfa' width="50%" height="100%"> | <video src='https://github.com/secretkeylabs/xverse-web-extension/assets/18606335/12fc70b9-fff7-4fd1-bfa8-2b65ca2373f6' width="50%" height="100%"> |

# ✅ Review checklist
Please ensure the following are true before merging:

- [ ] Code Style is consistent with the project guidelines.
- [ ] Code is readable and well-commented.
- [ ] No unnecessary or debugging code has been added.
- [ ] Security considerations have been taken into account.
- [ ] The change has been manually tested and works as expected.
- [ ] Breaking changes and their impacts have been considered and documented.
- [ ] Code does not introduce new technical debt or issues.

### CODE REVIEW

File: src/app/screens/settings/lockCountdown/index.tsx [hunk 1]
```diff
@@ -61,6 +62,14 @@ const TimerIcon = styled.img((props) => ({
   marginRight: props.theme.spacing(12),
 }));
 
+const getLabel = (period: number) => {
+  if (period < 60) {
+    return `${period} minutes`;
```
Comment (dhriaznov) [inline src/app/screens/settings/lockCountdown/index.tsx:67]: Could you also use the translation keys here?

**Comment by dhriaznov** [comment]
It looks like currently the 15 minutes option timer icon has opacity set to `1`, but all other icons have `0.8`. I took a look at figma and from what I see it should be dynamic, so that the active option icon should have an opacity `1` and all other icons should have `0.8`

https://www.figma.com/file/FCk8jDMsC8UEfPSWnqewLY/Xverse-%E2%80%94-Extension-V2?node-id=11693%3A13820&mode=dev

**Comment by fedeerbes** [comment]
> It looks like currently the 15 minutes option timer icon has opacity set to `1`, but all other icons have `0.8`. I took a look at figma and from what I see it should be dynamic, so that the active option icon should have an opacity `1` and all other icons should have `0.8`
> 
> https://www.figma.com/file/FCk8jDMsC8UEfPSWnqewLY/Xverse-%E2%80%94-Extension-V2?node-id=11693%3A13820&mode=dev

you're right, thanks! i took them from mobile and i didn't check that detail. will make the change

**Review by dhriaznov — APPROVED** [review]
LGTM

**Pull request merged**